tests/Unit/ControlSystem/Test_UpdateFunctionOfTime.cpp

```diff
@@ -208,14 +214,14 @@ void test_update_aggregator() {
 #endif
   aggregator.insert("California", DataVector{5.0}, 6.0,
                     signals.at("California"), 1.0);
-  CHECK(aggregator.is_ready());
+  CHECK(aggregator.is_ready(active_map));
```

Comment:
Before the call to `.insert` can you edit the `active_map` to turn `California` off and check that the aggregator is ready, and then turn `California` back on for the rest of the test? Just so we test dynamically turning things on and off.

Reply:
I think I did the requested thing right?? 🤞 Let me know if any other changes would be useful to help test :)

Reply:
Sorry, this was my bad for not being specific. Could you do
```cpp
#ifdef
...
#endif
active_map.at("California") = false;
CHECK(aggregator.is_ready(active_map));
active_map.at("California") = true;
CHECK_FALSE(aggregator.is_ready(active_map));
aggregator.insert("California", DataVector{5.0}, 6.0,
                  signals.at("California"), 1.0);
CHECK(aggregator.is_ready(active_map));
```
Since the control system will be turned off and not sending anything to the aggregator, it's important to check that we can not receive a measurement for a disabled control system and still be ready.

Reply:
Ah okay, thanks! Done :)